modules/local/rename_reads.nf

```diff
@@ -0,0 +1,91 @@
+process RENAME_READS {
```

Comment:
Wouldn't it be better if this became part of the cellrangermulti module? For consistency with the cellranger module and also for performance reasons (no download/reupload to cloud buckets). 

Or maybe this isn't as important anymore with the Fusion file system (it's anyway not important for shared file systems) -- then this could be a separate process for both cellranger and cellranger multi)

Reply:
I can try modifying and putting it there indeed.
I will first try directly in this PR, when finding a nice way, I can open a PR in the modules repo :)

Reply:
Discussion happening on: https://github.com/nf-core/modules/pull/4802

Reply:
Solved.